test.yml

```diff
@@ -0,0 +1,13 @@
+- hosts: localhost
```

Comment:
this is a reminder to delete this file eventually :)

Reply:
deleted!